mindsdb/api/executor/command_executor.py

```diff
@@ -164,18 +164,17 @@ def __init__(self, session, context=None):
         self.datahub = session.datahub
 
     @profiler.profile()
-    def execute_command(self, statement, database_name: str = None) -> ExecuteAnswer:
-        sql = None
-        if isinstance(statement, ASTNode):
-            sql = statement.to_string()
-        sql_lower = sql.lower()
+    def execute_command(self, statement: ASTNode, database_name: str = None) -> ExecuteAnswer:
+        sql: str = statement.to_string()
+        sql_lower: str = sql.lower()
 
         if database_name is None:
             database_name = self.session.database
 
-        if type(statement) is CreateDatabase:
+        statement_type = type(statement)
+        if statement_type is CreateDatabase:
```

Comment:
wouldn't be better to use isinstance(statement, CreateDatabase) ?

Reply:
we don't need to check inheritance, so `type` will be faster here